src/modules/Store.js

```diff
@@ -0,0 +1,49 @@
+class Store {
+  // LASTINDEX is the value of the last item index in the local storage.
+  static LASTINDEX = 0;
+
+  // get all to do lists from local stroage if local storage has a data.
+  // return lists of to do lists from the local storage.
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+      this.LASTINDEX = tasks.length;
+    }
+
+    return tasks;
+  }
+
+  // receive task as a parameter.
+  // add Task to local storage.
+  // update local storage.
+  // update LASTINDEX.
+  // return void.
+  static addTask(task) {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    this.LASTINDEX = tasks.length;
+  }
+
+  // receive task index as a parameter.
+  // fetch tasks from local storage keep it in array.
+  // delete task from arraly list.
+  // update local storage.
+  // update LASTINDEX.
+  // return void.
+  static removeTask(index) {
+    const tasks = Store.getTasks();
+    tasks.forEach((task, i) => {
+      if (task.index === Number(index)) {
+        tasks.splice(i, 1);
+      }
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    this.LASTINDEX = tasks.length;
+  }
```

Comment:
Please note that after removing tasks from your ToDO, the index should remain unique. i.e If I have 3 tasks and I remove the second task, the index should be 1 and 2, not 1 and 3
![Screenshot from 2023-05-05 13-10-13](https://user-images.githubusercontent.com/44624138/236455259-1accf7ea-e67b-468d-9923-f202047bbf37.png)

Reply:
Please no that all tasks should have their `completed` set to `false` by default, and only set to true when the box has been checked. The above image shows otherwise Kindly fix it 👍🏽.

Reply:
Please ensure that after editing a task and reloading the page, the task maintains the new description not the old 😉.

Reply:
thanks I forget to update the index.